carbon-design-system/carbon
# Added trap focus and focus redirection

Closes #14691

Enhanced the ActionableNotification component by implementing trap focus functionality, alongside the addition of a tabIndex attribute to the parent div. This enhancement ensures that when a user receives a notification with the hasFocus prop set to true, the focus is correctly directed to the component.

#### Changelog

**New**

- Added the trap focus functionality;
- Added `tabIndex` in the parent div.

#### Testing / Reviewing

- Check if the focus is trapped in the notification
- Check if the focus is working correctly in the `TestFocus` variant 
____________

- [ ] Delete the `TestFocus` variant
